dsa-23au/java-dsa/sorting/src/test/java/dev/codewithfriends/MergeSortTest.java

```diff
@@ -0,0 +1,36 @@
+package dev.codewithfriends;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class MergeSortTest {
+
+    @Test
+    public void testMergeSortWithEmptyArray() {
+
+        Integer[] inputArray = {};
+        Integer[] expectedArray = {};
+
+        MergeSort.mergeSort(inputArray);
+        assertArrayEquals(expectedArray, inputArray);
+    }
+
+    @Test
+    public void testMergeSortWithSingleElementArray() {
+
+        Integer[] inputArray = {8};
+        Integer[] expectedArray = {8};
+
+        MergeSort.mergeSort(inputArray);
+        assertArrayEquals(expectedArray, inputArray);
+    }
+
```

Comment:
The JUnit tests are easy to follow and understand. He does three different tests with an empty array, an array with a single element, and then an array that has multiple elements. I like the assertArrayEquals method, it is easier to compare arrays rather than loop through them.  This will help with our final project because we will have to test the methods that we write.